cluster/kes.go

```diff
@@ -162,17 +163,21 @@ path "kv/%s/*" {
 
 func getNewKesDeployment(deploymentName string, kesSecretsNames map[string]string) appsv1.Deployment {
 	kesReplicas := int32(1)
+	port, _ := strconv.Atoi(getKesRunningPort())
+	kesMTlsAuth := getkesMTlsAuth()
+	kesConfigPath := getKesConfigPath()
+	kesCommand := fmt.Sprintf("for i in {1..5}; do sleep 3; kes key create app-key -k && break || sleep 1; done & kes server --config=%s --mtls-auth=%s", kesConfigPath, kesMTlsAuth)
```

Comment:
why do we loop 5 times  ?

Reply:
We try 5 times to create the initial kes key, 5 its an arbitrary number, if we success the first time then we stop trying.